coalib/parsing/Globbing.py

```diff
@@ -384,15 +377,12 @@ def iglob(pattern):
     :return:        Iterator that yields all file names that match pattern
     """
     for pat in _iter_alternatives(pattern):
-        pat = os.path.expanduser(pat)
-        pat = os.path.normcase(pat)
+        pat = os.path.normcase(os.path.expanduser(pat))
```

Comment:
I often wonder why we don't use `path.py`. After `from path import Path` we could do such nice things like:

```python
pat = Path(pat).expanduser().normcase()
```